timemachine/cpp/src/tibd_exchange_move.cu

```diff
@@ -46,13 +43,12 @@ TIBDExchangeMove<RealType>::TIBDExchangeMove(
           num_proposals_per_move,
           interval,
           batch_size,
-          round_up_even(TIBD_TRANSLATIONS_PER_STEP_XYZXYZ * num_proposals_per_move)),
+          TIBD_TRANSLATIONS_PER_STEP_XYZXYZ * num_proposals_per_move),
       radius_(static_cast<RealType>(radius)), inner_volume_(static_cast<RealType>((4.0 / 3.0) * M_PI * pow(radius, 3))),
       d_rand_states_(DEFAULT_THREADS_PER_BLOCK), d_inner_mols_count_(1), d_identify_indices_(this->num_target_mols_),
       d_partitioned_indices_(this->num_target_mols_), d_temp_storage_buffer_(0), d_center_(3),
-      d_uniform_noise_buffer_(round_up_even(NOISE_PER_STEP * this->steps_per_move_ * this->batch_size_)),
-      d_targeting_inner_vol_(this->batch_size_), d_ligand_idxs_(ligand_idxs),
-      d_src_log_weights_(this->num_target_mols_ * this->batch_size_),
+      d_uniform_noise_buffer_(num_proposals_per_move), d_targeting_inner_vol_(this->batch_size_),
```

Comment:
just checking, `d_uniform_noise_buffer_` is now used only for deciding the targeted move, and the `mh` noise is inherited from the base class? 

also, is `this->steps_per_move_ * this->batch_size_ == num_proposals_per_move`?

Reply:
> just checking, d_uniform_noise_buffer_ is now used only for deciding the targeted move, and the mh noise is inherited from the base class?

Exactly

> also, is this->steps_per_move_ * this->batch_size_ == num_proposals_per_move?

Yup